region/lowernorfair/east.json

```diff
@@ -1656,11 +1656,11 @@
                     "heatProof",
                     "canHeatRun"
                   ]},
-                  "Super",
                   {"cost": [
-                    {"heatFrames": 1200}
+                    {"heatFrames": 400}
```

Comment:
I thought the room was symmetrical, is that not the case? The two frame counts are different right now. Also, only one side has the node,

Reply:
It's a bit tricky with the heatframes here since you need to kill the pirates in one direction and not the other but as far as I can remember this is just an artifact from the time I didn't consider hitboxing to achieve the charge. Making them both 400 for now.

Reply:
Oh... I thought that whole thing had been moved to the door's unlock, turns out it's in the link's unlock. That doesn't work, you have to unlock the door regardless of what side you're coming in from.

Then charging the spark wouldn't care about this.